### ISSUE TITLE
Duda sobre distintas cartas con mismos efectos

### ISSUE THREAD

**Issue by CGR0** [RodrigoToroIcarte/IIC2113-2023-1#204]
En el enunciado, en la parte de las cartas híbridas se mencionan 3 cartas que tienen los mismo efectos como Action, y en clases se mencionó que lo más probable era que tuvieramos que hacer una clase por carta, pero en el caso de las 3 cartas mencionadas anteriormente, las clases serían iguales a diferencia de los nombres, por lo que se tendría una repetición de código ahí. ¿Esta repetición da lo mismo o tengo que buscar otra forma de implementar estas tres cartas en una sola clase?

**Comment by RodrigoToroIcarte** [comment]
No, todo lo contrario. El mensaje es que NO hagan una clase por carta. Y el motivo es lo que dices. Te va a quedar código duplicado. 

![Screenshot from 2023-10-20 17-21-57](https://github.com/RodrigoToroIcarte/IIC2113-2023-1/assets/7000518/4398a273-fccb-4962-91b5-62329dc27284)

Esto no quita que existan formas de diseñar tu solución donde tienes una clase por carta sin código duplicado. Pero eso ya es entrar en una discusión media larga que no puedo explicar en este issue :)
